tools/verdict-back-ends/verdict-bundle/verdict-synthesis/src/test/resources/com/ge/verdict/synthesis/decimalCosts.xml

```diff
@@ -1,5 +1,5 @@
 <cost-model>
     <cost component="A" defense="A" dal="1">4.2</cost>
-    <cost component="B" defense="B" dal="1">0.035</cost>
+    <cost component="B" defense="B" dal="1">1.035</cost>
```

Comment:
Costs < 1 are no longer supported. All costs must be greater than the base scaling factor (1)